src/libs/antares/study/parts/hydro/hydromaxtimeseriesreader.cpp

```diff
@@ -128,9 +128,10 @@ bool HydroMaxTimeSeriesReader::read(const AnyString& folder, bool usedBySolver)
 {
     bool ret = loadDailyMaxPowersAndEnergies(folder, usedBySolver);
     copyDailyMaxEnergy();
+    hydro_.series->computeTSCount();
```

Comment:
`hydro_.series->computeTSCount();` : 
Please see **src/libs/antares/study/area/list.cpp**'s review for a comment on that matter.

Reply:
This method has been removed. We now compute the TS count every time it's necessary.